source/features/fork-source-link-same-view.tsx

```diff
@@ -37,7 +38,7 @@ async function getEquivalentURL(): Promise<string> {
 
 async function init(): Promise<void> {
 	// The link must always be updated/reset. This pattern ensures that the link is always updated and never fails through some conditions.
-	select<HTMLAnchorElement>(`[data-hovercard-url="/${getForkedRepo()!}/hovercard"]`)!.href = await getEquivalentURL();
+	select(`a[data-hovercard-url="/${getForkedRepo()!}/hovercard"]`)!.href = await getEquivalentURL();
```

Comment:
Huh they added support for concatenated strings? 😮 This change can probably be done to more places

Reply:
Hmm I don't know of this is only supported recently, I just thought this looks better.

Reply:
I think we did it where we were able to already

Reply:
There are more places. For some reason sometimes it works, others it doesn't 🤷‍♂️ 

```js
❯ rg select\<
source/options.tsx
212:                    const input = select<HTMLInputElement>('#' + feature)!;

source/features/suggest-commit-title-limit.tsx
15:     const inputField = select<HTMLInputElement>(fieldSelector)!;

source/features/highest-rated-comment.tsx
70:     const {hash} = select<HTMLAnchorElement>('a.js-timestamp', bestComment)!; // TODO: Drop generic once TypeScript sorts it out, it shouldn't be needed
```